test/e2e/framework/pv/pv.go

```diff
@@ -197,15 +199,15 @@ func DeletePersistentVolumeClaim(c clientset.Interface, pvcName string, ns strin
 // phase value to expect for the pv bound to the to-be-deleted claim.
 func DeletePVCandValidatePV(c clientset.Interface, ns string, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume, expectPVPhase v1.PersistentVolumePhase) error {
 	pvname := pvc.Spec.VolumeName
-	Logf("Deleting PVC %v to trigger reclamation of PV %v", pvc.Name, pvname)
+	e2elog.Logf("Deleting PVC %v to trigger reclamation of PV %v", pvc.Name, pvname)
 	err := DeletePersistentVolumeClaim(c, pvc.Name, ns)
 	if err != nil {
 		return err
 	}
 
 	// Wait for the PV's phase to return to be `expectPVPhase`
-	Logf("Waiting for reclaim process to complete.")
-	err = WaitForPersistentVolumePhase(expectPVPhase, c, pv.Name, Poll, PVReclaimingTimeout)
+	e2elog.Logf("Waiting for reclaim process to complete.")
+	err = framework.WaitForPersistentVolumePhase(expectPVPhase, c, pv.Name, framework.Poll, framework.PVReclaimingTimeout)
```

Comment:
I think it would make sense to move `framework.PVReclaimingTimeout` into `/test/e2e/framework/pv` (e2epv) as well.
This one is used by e2epv and e2e/storage/volume_provisioning.go.